vm/os-windows.cpp

```diff
@@ -296,6 +296,50 @@ void handle_ctrl_c() {
   SetConsoleCtrlHandler(factor::ctrl_handler, TRUE);
 }
 
+const int ctrl_break_sleep = 10; /* msec */
```

Comment:
Actually, the delay is not about the reaction speed, it's about catching the Ctrl-Break combo in its depressed state. It turned out to be easy to miss with longer delays. It has been established earlier in this discussion that polling is extremely lightweight and doesn't affect anything.

The no-polling patch was implemented based on Raw Input API, but didn't work, because the watcher stole all keyboard input, so commands like `readln` became impossible in the listener.

@bjourne: "I think the polling variant is the best solution. It would get awfully tricky to have one thread that takes all keyboard input and tries to delegate to the main thread. "